app/assets/javascripts/govuk_publishing_components/components/header.js

```diff
@@ -0,0 +1,5 @@
+// This component relies on JavaScript from GOV.UK Frontend
+// = require govuk/components/header/header.js
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+window.GOVUK.Modules.Header = window.GOVUKFrontend
```

Comment:
If we want to move forward with a backwards-compatible approach, I suggest we use symlinks so we avoid having duplicate files (e.g. `ln -s ~/govuk/govuk_publishing_components/app/assets/javascripts/govuk_publishing_components/components/layout-header.js ~/govuk/govuk_publishing_components/app/assets/javascripts/govuk_publishing_components/components/header.js`)

Reply:
That seems like a better approach but can we guarantee that path is correct in all environments? Small thing, but those paths don't match my local system, so it wouldn't work for me locally - although I'm much more concerned about production environments. I think a copy might be safer - this is only a temporary solution after all.

Reply:
OK. Can we add a deprecation warning at the top of these files then? – might prevent people from changing them.

Reply:
Symbolic links can be relative - and using relative links would mean that it doesn't reference a path outside of the govuk_publishing_components folder.

Reply:
Will add a warning, good idea.

Reply:
Indeed @injms – my example was rudimentary, I was hoping for a relative reference in the code